src/XenobladeX/Mods/BladeMoreTickets/rules.txt

```diff
@@ -2,8 +2,12 @@
 titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
 name = "More Reward Tickets"
 path = "Xenoblade Chronicles X/Mods/BLADE/More Reward Tickets"
-description = Increase the Tickets gained with Squad tasks and missions. Uncapped presets ignore max tickets.
-version = 4
+description = Increase the Tickets gained with Squad tasks and missions.|Uncapped presets ignore max tickets.
+version = 5
+
+[Default]
+$mult = 1
```

Comment:
probably should add a preset like "regular ticket rate" so that this 1 gets used